lightning/src/ln/peer_handler.rs

```diff
@@ -543,7 +544,8 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 		SecretKey::from_slice(&Sha256::from_engine(ephemeral_hash).into_inner()).expect("You broke SHA-256!")
 	}
 
-	/// Indicates a new outbound connection has been established to a node with the given node_id.
+	/// Indicates a new outbound connection has been established to a node with the given node_id
+	/// and an optional remote network address.
```

Comment:
Is this okay? @valentinewallace

Reply:
This should describe when a user should pass a `remote_network_address`, what the reason is, and what considerations they should think about when deciding whether or not to include it.